nodes/config.js

```diff
@@ -132,7 +149,11 @@ module.exports = function(RED) {
       return;
     }
 
-    const callback = callbacks[req.body.content["deviceId"]];
+    if(req.body.content["deviceId"] != null) {
+      var callback = callbacks[req.body.content["deviceId"]];
+    } else if (req.body.content["deviceId"] == null && req.body.content["name"] == "mode") {
```

Comment:
Avoid redundancy condition
```suggestion
    } else if (req.body.content["name"] == "mode") {
```